tests/integration_tests/framework/docker.py

```diff
@@ -161,7 +160,6 @@ def run_manager(image, resource_mapping=None):
     command += [image]
     manager_id = subprocess.check_output(command).strip()
     execute(manager_id, ['cfy_manager', 'wait-for-starter'])
-    upload_mock_license(manager_id)
```

Comment:
Why are we not uploading the license here now?

Reply:
i guess we could. It fits for me better to upload it in the other fixture, because that fixture is about keeping on-manager storage consistent, and i feel the license is part of that. While this function is just about starting the container.

I added a new commit that renames & documents the fixture i mean (`prepare_manager_storage`)

Reply:
Sounds reasonable.